json/test/index.js

```diff
@@ -12,6 +12,10 @@ test('lookup project codes', function (t) {
   proj = codes.lookup(3819)
   t.equal(proj.name, 'GCS_HD1909')
 
+  // vertical codes
+  proj = codes.lookup(3855)
+  t.equal(proj.name, 'EGM2008_Geoid')
```

Comment:
Please change this name to "EGM2008_height" as we are going to change the name to match EPSG definition.